### DISCUSSION TITLE
Some IFC properties (area, volume) not reaching iTwin Exporter for Datasmith

### DISCUSSION THREAD

**Discussion by Hendawy97** [iTwin/itwinjs-core#3663] [Q&A]
recently after using the i-model for datasmith exporter I found that the data like area and volume were lost during exportation and i couldn't find them in unreal.

**Comment by MichaelBelousov** [comment]
Hi @Hendawy97, where we left off discussing this, I wasn't sure how your synchronization workflow differed from my test.

Is what you did exactly:

1. use synchro pro's native IFC2x3 export to get an ifc file from your model (and confirmed properties are discoverable in synchro pro and the ifc file)
2. synchronize that IFC file to an iModel (and confirm the properties are discoverable in the iModel, you can check in the iTwin Exporter for Datasmith)
3. open the file in iTwin Exporter for Datasmith and export it to an Unreal Datasmith file using the "all metadata" option.
4. find there is no metadata in the unreal actors?

I would also say, please check the components of the actors in your exported datasmith scene, Epic stores the datasmith metadata scene on the components sometimes instead of on the actors themselves.

I have not been able to reproduce the losing of those properties given the steps above. Please let me know how your synchronization process differed.

**Reply by MichaelBelousov** [reply]
@Hendawy97 clarified that the issue occurs like so:

1. import a revit file into synchro pro
2. upload synchro pro to synchro control to connect to iModel
3. load the iModel in the iTwin Exporter for Datasmith, it should have the volume/area properties
4. export for datasmith with "all metadata", the properties should be gone

I will try to reproduce
